docs/dev/fundamentals/testnet.md

```diff
@@ -29,9 +29,9 @@ All the existing SDKs for Ethereum will work out of the box and your users will
 
 The only other place where using zkSync SDK is required is during contract deployment. This can be easily done through our hardhat plugin.
 
-## Hello World on zkSync
+## Quickstart tutorial on zkSync
```

Comment:
It's a tutorial, we don't have to explicitly add it to the heading, hence "Quickstart on zkSync", sounds like it.